nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -136,62 +161,8 @@ in
         type = types.package;
         default = pkgs.slurm.override { enableX11 = ! cfg.enableSrunX11; };
         defaultText = literalExpression "pkgs.slurm";
-        example = literalExpression "pkgs.slurm-full";
-        description = ''
-          The package to use for slurm binaries.
-        '';
-      };
-
-      controlMachine = mkOption {
```

Comment:
I would prefer to make a hard change here for the following reason. It may lead to confusion when both, legacy and `settings` are used. Which one should be used if it affects one an the same setting? It would make the module more messy trying to handle both. The conversion of a config file should be rather straight forward. 

We could use `mkRenamedOptionModule` where applicable and `mkRemovedOption` for the rest?